core/beanstalk_userdata_example.txt

```diff
@@ -0,0 +1,54 @@
+Content-Type: multipart/mixed; boundary="===============5189065377222898407=="
```

Comment:
I wouldn't put these files here, I propose to create a test_data directory in the root dir of the repo and move them there